frontend/pages/admin/IntegrationsPage/cards/MdmSettings/VppPage/components/EditTeamsVppModal/EditTeamsVppModal.tsx

```diff
@@ -203,8 +203,9 @@ const EditTeamsVppModal = ({
           Edit teams for <b>{currentToken.org_name}</b>.
         </p>
         <p>
-          If you remove a team, the App Store apps will be removed from that
-          team. They won&apos;t be uninstalled from hosts.
+          If you remove a team, App Store apps will be removed from that team,
+          and policy automations referencing those apps will be removed.
```

Comment:
```suggestion
          and policies that trigger automatic install of these apps will be removed.
```

Same as above. Aligning language to one we already use in the software title details when referring to policies that trigger automatic install.